parm/config/config.base.emc.dyn

```diff
@@ -180,15 +233,15 @@ export gfs_cyc=@gfs_cyc@ # 0: no GFS cycle, 1: 00Z only, 2: 00Z and 12Z only, 4:
 # GFS output and frequency
 export FHMIN_GFS=0
 
-export FHMAX_GFS_00=384
-export FHMAX_GFS_06=384
-export FHMAX_GFS_12=384
-export FHMAX_GFS_18=384
+export FHMAX_GFS_00=${FHMAX_GFS_00:-384}
+export FHMAX_GFS_06=${FHMAX_GFS_06:-384}
+export FHMAX_GFS_12=${FHMAX_GFS_12:-384}
+export FHMAX_GFS_18=${FHMAX_GFS_18:-384}
```

Comment:
I have always wondered, is there ever a practical reason why a user would set different forecast lengths for different cycles?

Reply:
I believe there have been times in the past when 00z or 00z & 12z have been run longer than other cycles.